packages/react-theming/src/examples/compose/index.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import { initalize } from './../..';
+import { ThemeProvider } from './../../components/ThemeProvider/ThemeProvider';
+import { compose } from './../../compose';
+import { Variant } from './../../variant';
+import { ITheme, IColorRamp } from '../../theme.types';
+
+initalize();
```

Comment:
Just initial thought when I glanced at this that I wasn't sure what this was initializing.